tests/e2e/support/utils/runtimeFs.ts

```diff
@@ -0,0 +1,93 @@
+import fs from 'fs'
+import path from 'path'
+import { config } from '../../config'
+
+// max file creation size is 10GB
+export const MAX_FILE_SIZE = Math.pow(1024, 3) * 10
+
+export const getBytes = (fileSize: string): number => {
+  fileSize = fileSize.replace(/\s/g, '').toLowerCase()
+  const size = parseFloat(fileSize.match(/([\d.]+)/)[0])
+  const type = fileSize.match(/[kKmMgGbB]{1,2}/)[0]
+
+  let sizeInbytes = size
```

Comment:
for this file name too

Reply:
I want to relate it with the node `fs` library. helper for files/folders
Suggest me some names.